onnx/backend/test/runner/__init__.py

```diff
@@ -175,17 +205,15 @@ def _prepare_model_data(self, model_test):  # type: (TestCase) -> Text
             # On Windows, NamedTemporaryFile can not be opened for a
             # second time
             download_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Let's put the creation and deletion of the tmp file into the _download_model? (I think it's ok to create tmp files couple times)

Reply:
Sure, and move the "os.remove" into it as well